_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+This is a special notification type available for media apps since Android (5.0) Lollipop. In Oreo (8.0), it could colour-match its theme to the cover art of the music playing. Since Android 10, it has a SeekBar built in. It is a powerful API removing the need to use a whole bunch of custom code to achieve what was. Easier to maintain and improve. Win!
 
 ### Android 9
 
-First of all, we say "sorry" to our users on Android 9 and below. The progress bar is gone. As we've said above, it will only be shown on Android 10 and above. We're a small team and can't maintain code for the old and new types of notifications simultaneously, so the old one has to go. And not just that; the old notification isn't even supported Android 11 onwards. Bye-bye RemoteView notifications. We shall miss you! :'(
-
-But you get a lot in exchange for that sacrifice!
+Android 9 is not Android 10, and there is now only a progress bar on 10. It makes sense to keep things managable by our small team. The old notification isn't even supported on Android 11 onwards. RemoteView notifications are out, MediaStyle is in.
+Your sacrifice is not in vain:
 
 ### New features
 
-You get five CUSTOMISABLE - you read it right! - **CUSTOMISABLE** action buttons to control playback. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, and you can choose which button should perform which player action. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing!
+Button up or button down, you get five **customisable** action buttons to control playback. Batteries not included. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, from where you can pick actions and the buttons to carry out your bidding. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing! The decadency.
 
-Then, you can choose which three of these five buttons show in the compact notification (remember that little arrow you use to expand/contract the notification?).
+Three of these five buttons can be shown in the compact notification (remember that little arrow you use to expand/contract the notification?).
 
-For users on **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As we've noted above, MediaStyle notifications adapt to the colours of the cover art, which looks just plain awesome.
+For the betrodden of **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As noted above, MediaStyle notifications adapt to cover art colours. Big.
 
-For users on **Android 10+**: Seekbar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and have fun seeking!
```

Comment:
It isn't "seekbar", it is "SeekBar". We know the old version had a progress bar, because, it was mentioned already. "that was just a visual indicator" isn't doing anything.
"Have fun seeking" is going somewhere. Almost.

Reply:
> It isn't "seekbar", it is "SeekBar".

Okay. You were skeptical about us mentioning "RemoteView" above but now you insist on us using the proper object name in the code?

> We know the old version had a progress bar, because, it was mentioned already. "that was just a visual indicator" isn't doing anything.

It _is_ doing something, namely, contrasting between the old and the new notifications. You can seek on the new one, but not on the old one. New feature being highlighted. That is one of the goals of the blog post.

> "Have fun seeking" is going somewhere. Almost.

I had originally made a Harry Potter Quidditch reference, but @Poolitzer removed it because he is an old and boring person. He is not 'hip' at all. ಠ╭╮ಠ

Reply:
So I finally found a way to see what comment pinged me since GitHub definitly isn't made for whatever you two are doing, and this is the message. Wow. Thanks. No one likes Harry Potter anyway.

Reply:
ಠ▃ಠ